pulp_container/app/utils.py

```diff
@@ -309,3 +314,39 @@ def get_content_data(saved_artifact):
         raw_data = file.read()
     content_data = json.loads(raw_data)
     return content_data, raw_data
+
+
+def include(x, patterns):
+    """
+    Checks if any item from `patterns` matches x, meaning it should be included as a remote repo.
```

Comment:
We should make these docstrings more generic. We do not know how the functions are going to be used. The caller is the one that decides the purpose of the functions. For these one-line functions, it might be worth omitting the docstring too. Also, we do not care only about remote repositories in this context.